content/pandas/concepts/groupby/groupby.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: 'GroupBy'
+Description: 'The GroupBy object is returned by calls to `.groupby()` on a Series or DataFrame, used to apply functions to, or aggregate groups.'
```

Comment:
```suggestion
Description: 'The GroupBy object is returned by calls to `.groupby()` , a function that reassembles the data into distinct groups, on a Series or DataFrame, often for aggregation.'
```

Reply:
I don't think this suggested phrasing is all that good, but I think it is more directly stating what groupby is about. I think 'used to apply functions to' is too abstract.